Resources/public/js/views/ez-contenteditformview.js

```diff
@@ -35,6 +35,7 @@ YUI.add('ez-contenteditformview', function (Y) {
 
         initializer: function () {
             this._setFieldEditViews();
+            this.after('versionChange', this._setFieldEditViews);
```

Comment:
in some others similar cases, it's done with a `setter` (for instance in https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/js/views/ez-editactionbarview.js#L133). Also by calling `_setFieldEditViews` you are going to recreate the whole field edit views list while you just want to update the version (and the field).